ci/axis/miniforge-cuda-driver.yml

```diff
@@ -14,9 +14,7 @@ DOCKER_FILE:
 CUDA_VER:
   - 11.5
   - 11.4
-  - 11.3
```

Comment:
CUDA 11.3 is associated with driver 465 (see table 3 [here](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html)). so any mentions of driver 465 in this file should also be removed.